src/main/java/io/github/sefiraat/slimetinker/items/componentmaterials/cmfactories/CMCore.java

```diff
@@ -996,15 +996,15 @@
                                                 CMTraits.PROP_PLATES,
                                                 SupportedPluginsManager.CORE_NOTE,
                                                 "Band",
-                                                "Durability loss + 200%. Damage reduced between 1-25%",
+                                                "Durability loss + 300%. Damage reduced between 1-25%",
```

Comment:
For this one the DurabilityEvent#plateBrass method is actualyl the one that is wrong. +200% is intended, I simply screwed up number. Can you undo this and either change the method `friend.setDurabilityMod(friend.getDurabilityMod() + 3);`. Good catch though, you'd have no way to know which way was correct :) !